auction9-frontend/src/app/data-table/data-table-details/data-table-details.component.html

```diff
@@ -39,6 +39,6 @@ <h4>Number of bids:
   	<mat-form-field class="input-bid">
 	    <input matInput placeholder="Offer must start from: {{auction?.price + 1}} din" value="{{auction?.price + 1}}" min="{{auction?.price + 1}}" required>
   	</mat-form-field>
-    <button mat-raised-button color="primary">Bid</button>
+    <button mat-raised-button color="primary" [disabled]="auction?.status === 'ACTIVE' ? null : 'disabled'">Bid</button>
```

Comment:
Is it maybe enough to put only condition because it will return boolean value. example: [disabled]="auction?.status === 'ACTIVE'"

Reply:
It has to be this way as it's written to work correctly